.github/workflows/canary-integration-test.yml

```diff
@@ -1207,3 +1207,52 @@ jobs:
       if: failure() && github.event_name == 'pull_request'
       uses: mxschmitt/action-tmate@v3
       timeout-minutes: 60
+
+  csi-hostnetwork-disabled:
```

Comment:
Instead of adding a new test, what if we just change existing tests to run without host networking? We want to make this the default anyway. And do we already have a test that restarts the CSI plugins so we could verify that it's working?

Reply:
Sure we can keep it, that's fine. I was just asking since it's a general pattern to use the existing test suites and enable different settings for the different suites, instead of creating new tests for every setting.

Reply:
Once pod networking becomes stable will remove this one as the default will be pod networking and also I will take a look at it later how to test both pod network and host network in the same test suite by changing multiple settings instead of creating new tests for every setting.